openimaj/src/test/java/science/aist/imaging/service/openimaj/imageprocessing/transformer/ColourSpace2ChannelTypeTransformerTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 the original author or authors.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package science.aist.imaging.service.openimaj.imageprocessing.transformer;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * <p>Test class for {@link ColourSpace2ChannelTypeTransformer}</p>
+ *
+ * @author Christoph Praschl
+ * @since 1.1
+ */
+public class ColourSpace2ChannelTypeTransformerTest {
+
+    @Test
+    public void testTransformTo() {
+        // given
+
+        // when
+
+        // then
+    }
```

Comment:
Nah.

Reply:
Fixed in [739c19f](https://github.com/FHOOEAIST/imaging/pull/14/commits/739c19fa2d18dad40ebac4676254c36ff07279f0)